<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="POL-6MXJDB3" failures="1" tests="3" name="PlaywrightSeries" time="0.044" errors="0" timestamp="2024-10-04T17:07:04 CEST">
  <testcase classname="com.dydko.lesson084_105.TestNG123" name="testcase2" time="0.004">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <testcase classname="com.dydko.lesson084_105.TestNG123" name="testcase3" time="0.0">
    <skipped/>
  </testcase> <!-- testcase3 -->
  <testcase classname="com.dydko.lesson084_105.TestNG123" name="testcase4" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.dydko.lesson084_105.TestNG123.testcase4(TestNG123.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase classname="com.dydko.lesson084_105.TestNG123" name="testcase1" time="0.0">
    <ignored/>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- PlaywrightSeries -->
